### PR TITLE
Added Snackbar for Pause/Resume

### LINKED ISSUE #72: Add Snackbar when a profile is paused/resumed
**Describe the feature you'd like**
Add a snackbar whenever the user pauses or resumes a profile.
Message to be shown: "Profile Name" is Paused/Resumed
Snackbar time: Short

Note: You cannot pass/store the view reference from MainFragment in the profileListAdapter. 

**Would you like to work on the issue?**

- [ ] Yes
- [ ] No
- Other: Open for all

### PR DESCRIPTION
**Pull request by coder2699**
Fixes #72 
<table>
   <tr>
<td></td>
<td></td>
</tr>
</table>

### CODE REVIEW

**Comment by coder2699** [comment]
@rob729 I am adding the screenshots, it will take 5 minutes though

**Comment by coder2699** [comment]
> @rob729 I am adding the screenshots, it will take 5 minutes though

Done

File: app/src/main/java/com/rob729/quiethours/Adapter/ProfileListAdapter.kt [hunk 1]
```diff
@@ -63,7 +65,15 @@ class ProfileListAdapter(
             Color.rgb(255, 143, 0),
             Color.rgb(216, 27, 96)
         )
-
+        private fun SnackBar(cardView: CardView, message: String) {
```
Comment (rob729) [inline app/src/main/java/com/rob729/quiethours/Adapter/ProfileListAdapter.kt]: Move the function to Utils and take view, message,  as parameter.
Don't take cardView type as a paramter.

Comment (coder2699) [reply app/src/main/java/com/rob729/quiethours/Adapter/ProfileListAdapter.kt]: Done

**Comment by coder2699** [comment]
@rob729 Please review my PR, before the Deadline Ends (12 noon)

File: app/src/main/java/com/rob729/quiethours/Adapter/ProfileListAdapter.kt (hunk 1 above)
Comment (rob729) [reply app/src/main/java/com/rob729/quiethours/Adapter/ProfileListAdapter.kt]: My bad @coder2699. I forgot to mention that snackbar length was also supposed to be taken.
Make the default value of snackbar length to short.
Move the function to Utils.kt and rename the function to showSnackBar()
Also, use this function in MainFragment(in swipe to delete and delete all profiles) and NewProfileFragment

**Review by rob729 — CHANGES_REQUESTED** [review]
@coder2699 Rename the function to showSnackbar() and replace the same function in MainFragment and NewProfileFragment wherever snackbar is shown.

**Comment by coder2699** [comment]
@rob729 I am making the changes, if it's working fine can you please transfer the points?

**Review by rob729 — APPROVED** [review]

**Comment by rob729** [comment]
give @coder2699 5 points

**Pull request merged**